app/controllers/application_controller.rb

```diff
@@ -26,6 +26,10 @@ def render_not_found
     render file: "#{Rails.root}/public/404.html", status: :not_found
   end
 
+  def render_hidden(message)
+    render 'pages/hidden', locals: { message: message }
+  end
```

Comment:
Hidden pages should be served with `status: :unauthorized`, because that's exactly what's going on: you're not authorized to view it (whereas a moderator could).